### ISSUE TITLE
admins should not be able to delete hospitals

### ISSUE THREAD

**Issue by KelseyDH** [pathwaysmedical/frasernw#252]

**Comment by KelseyDH** [comment]
Recent email:

```
1.   Administrators still have the ability to create a new clinic/hospital outside of their own Division's "cities encompassed".  I believe this was a "bug" that needed fixing. 

2.  I  noticed recently that when I was logged in as a Burnaby administrator, I went into Admin>hospitals>, which displayed all of the hospitals on Pathways.  I noted that the Burnaby administrator had the ability to edit/delete a few hospitals which were outside of their "cities encompassed" (although they appeared to be hospitals that appear to be associated with Divisions not yet implemented).  Even so, I don't believe they should have the ability to delete these hospitals.

```

**Comment by KelseyDH** [comment]
Notify Mary Anne when this issue is finished.

**Comment by dhMuse** [comment]
Closed in favour of REQ#50 and REQ#51.
